.travis.yml

```diff
@@ -11,6 +11,15 @@ go:
 - 1.6
 #- tip # NOTE: disabled - see https://github.com/travis-ci/gimme/issues/38
 
+before_cache:
+  - rm -rf $GOPATH/src/github.com/GoogleCloudPlatform/golang-samples
```

Comment:
I'm not sure. To be honest, I don't really understand the YAML syntax. I'll try removing them.